impeller/entity/contents/content_context.cc

```diff
@@ -352,11 +354,21 @@ std::shared_ptr<Texture> ContentContext::MakeSubpass(
   if (context->GetCapabilities()->SupportsOffscreenMSAA() && msaa_enabled) {
     subpass_target = RenderTarget::CreateOffscreenMSAA(
         *context, texture_size, SPrintF("%s Offscreen", label.c_str()),
-        RenderTarget::kDefaultColorAttachmentConfigMSAA, std::nullopt);
+        RenderTarget::kDefaultColorAttachmentConfigMSAA  //
+#ifndef FML_OS_ANDROID
```

Comment:
Mind adding another comment nearby about how keeping the stencil attachment is just to keep PSO variants to a minimum on Android (similar to the one below)?

Reply:
Done